website/build.gradle

```diff
@@ -104,3 +106,32 @@ task testWebsite(type: Exec) {
 task preCommit {
   dependsOn testWebsite
 }
+
+task mergeWebsite << {
+  /* Disabled for now, for testing
+  exec {
+    executable 'git'
+    args 'checkout', 'asf-site'
+  }
+  copy {
+    from contentBuildDir
+    into contentRepoDir
+  }
+  exec {
+    executable 'git'
+    args 'add', contentRepoDir
+  }
+  exec {
+    executable 'git'
+    args 'commit', '-m', 'Update website ' + new Date().format('yyyy/MM/dd HH:mm:ss')
+  }
+  exec {
```

Comment:
It would be useful for testing to split out everything before this to a separate task. Then we could run the prerequisite task locally to verify behavior or repro bugs.

Reply:
Using org.ajoberstar.git-publish which defines a :gitPublishCopy task which creates all the files without commit or push.